node-module-system/calculate.js

```diff
@@ -0,0 +1,19 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+switch (process.argv[3]) {
```

Comment:
You repeat `process.argv` repeatedly. It would be nice to extract it and only do it once.

It would also be better organized if you took care of the user input in once place and then passed variables of the proper type to your functions.